template.yaml

```diff
@@ -823,7 +865,7 @@ Resources:
             - IsDev
             - !Join [ ".", [ !Ref Developer, !FindInMap [ EnvironmentConfiguration, !Ref Environment, domainName ] ] ]
             - !FindInMap [ EnvironmentConfiguration, !Ref Environment, domainName ]
-          COGNITO_CLIENT_ID: !Ref DeathNotificationClient
+          COGNITO_CLIENT_ID: !Ref DeathDwpNotificationClient
```

Comment:
Think I prefer DwpDeathNotificationClient for if there are other clients for death notifications then the constant part is all together (i.e., <clientName>DeathNotificationClient)